meta-dts-distro/recipes-dts/dts/dts/dts-functions.sh

```diff
@@ -266,15 +278,15 @@ board_config() {
           PROGRAMMER_EC="ite_ec"
           FLASHROM_ADD_OPT_DEPLOY="--ifd -i bios"
           if check_if_dasharo; then
-          # if v1.6.0 or older, flash the whole bios region, as per:
+          # if v1.7.2 or older, flash the whole bios region, as per:
```

Comment:
@TomaszAIR Why? I thought from 1.7.1 onwards we can just flash RW region, or not, @mkopec ?

Reply:
@macpijan yesterday we tested that with @mkopec and this was his idea to flash whole bios region.

Reply:
@macpijan Not since commit https://github.com/Dasharo/coreboot/commit/f71119b07e6c17b5f1b4077fd7342b110017d5f1 which changed the VBNV offset and thus made the old bootblock incompatible

Reply:
I will resolve this conversation and merge this branch to develop. @macpijan if you still think that we should do something with that flags, please start new thread in rel_v1.2.13 MR